README.rst

```diff
@@ -32,14 +32,14 @@ Then, construct the thermometer class:
 
   # Do one reading
   with busio.I2C(SCL, SDA) as i2c:
-    t = adafruit_mcp9808.MCP9808(i2c)
+      t = adafruit_mcp9808.MCP9808(i2c)
 
 Finally, read the temperature property.
 
 .. code-block:: python
 
-    # and print it out
-    print(t.temperature)
+  # and print it out
```

Comment:
Please indent this too. It should match the code around it.

Reply:
Hmm. Adding four spaces to the comment seems to cause the code block to ignore the spaces in both lines. Please ignore the amended pull request for now and let me look into how to fix that.

Reply:
Oh weird! Maybe the blocks should just be merged together? Thanks for figuring this out!

Reply:
I think that would be the simplest solution. A quick web search shows that this is something people have run into before and there doesn't appear to be an elegant solution. reST gobbles spaces when all lines have the same indent. I'll amend the pull request.